.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
Let's standardise how we do this across projects, [like so](https://github.com/stylelint/stylelint-config-standard/blob/708cb8a4b53b82fc0e2e982908c988a2b539541d/package.json#L41-L55):

```
// package.json
{
  "scripts": {
    "format": "prettier . --write",
    "lint:formatting": "prettier . --check",
    "lint:js": "eslint . --ignore-path .gitignore",
    "lint:md": "remark . --quiet --frail --ignore-path .gitignore",
    "lint": "npm-run-all --parallel lint:*",
    "release": "np",
    "test": "jest",
    "watch": "jest --watch"
  },
  "eslintConfig": {
    "extends": [
      "stylelint"
    ]
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "*.js": "eslint --cache --fix",
    "*.(js,md,yml}": "prettier --write"
  }
}
```

Integrating Prettier into ESLints floods our editors with squiggly red lines for unimportant formatting issues.

Reply:
Thank you for your review!

I made this change.

Reply:
Thanks for adding the linting run-script commands.

> Integrating Prettier into ESLint floods our editors with squiggly red lines for unimportant formatting issues.

Can you:

- replace `eslint-config-prettier` with a `npm run format` command
- move the eslint config into `package.json`

So that it is consistent with the other stylelint repos.

Like so:

```
// package.json
{
  "scripts": {
    "format": "prettier . --write",
    "lint:formatting": "prettier . --check",
    "lint:js": "eslint . --ignore-path .gitignore",
    "lint": "npm-run-all --parallel lint:*",
  },
  "eslintConfig": {
    "extends": [
      "stylelint"
    ],
    parserOptions: {
      ecmaVersion: 2019,
    },
     rules: {
	'jest/valid-expect': 'off',
     }
  }
}
```

Reply:
I moved the eslint configuration to package.json.

However, eslint-config-prettier is required. This prevents conflict between prettier and eslint. Specifically, it conflicts with "no-mixed-spaces-and-tabs".

Reply:
Sorry, my bad. I mixed up the prettier plugin with the prettier config.

We should move eslint-config-prettier into [eslint-config-stylelint](https://github.com/stylelint/eslint-config-stylelint), like we intend to do with the equivalent remark preset in [`@stylelint/remark-preset`](https://github.com/stylelint/remark-preset-lint-stylelint/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bc), as it's applicable to all stylelint repos.

We can do this in a follow-up pull request once we release a new version of eslint-config-stylelint, though.